src/util/interaction/out.ts

```diff
@@ -356,7 +356,12 @@ report.allProperties = function (data: any): any {
 };
 
 report.asDate = function (data: any): string {
-  return new Date(Date.parse(data)).toString();
+  const date = new Date(data);
+  if (formatIsJson()) {
+    return date.toJSON();
```

Comment:
I have noticed that the example from command spec uses ISO string for JSON. It also seems to be used commonly for Dates inside the JSON.